system/jlib/jexcept.cpp

```diff
@@ -1117,6 +1117,10 @@ NO_SANITIZE("alignment") void excsighandler(int signum, siginfo_t *info, void *e
     if (nested++)
         return;
 
+    //If the program is terminating then do not try and trace
+    if (!queryLogMsgManager())
+        return;
```

Comment:
it may not matter much, but should it be after the #if defined(NO_LINUX_SEH) block, where it turns off future signal handling?

Reply:
I don't think it will make much difference - this function will return immediately anyway.